StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/SheetViewController.swift

```diff
@@ -13,6 +13,14 @@ class SheetViewController: UIViewController {
 
     private static let cornerRadius: CGFloat = 20
 
+    // Indicates whether to enable sheet-specific logic.
+    //
+    // Due to `SheetViewController` being a subclass, and auth flow
+    // design constraints of dynamically presenting panes either
+    // as sheets or full-screen, we need this boolean to handle
+    // both states (sheet or full-screen).
+    private let presentAsSheet: Bool
```

Comment:
non-blocking thought: right now I think `presentAsSheet` how you have it is easy enough to read. But if there's more planned logic that will be controlled by this thoughts on making it an enum for readability? so in places that need to check it it could read as:

`if sheetPresentationStyle == .fullscreen {` or `... == .sheet`

Reply:
Thank you for the suggestion, I'll change it to use an enum

My initial thought, because my default mantra is to always do the simplest possible thing, was to keep the boolean `presentAsSheet` (because its literally simpler than an enum [in terms of pure "computer logic"], and we only know of one extra/new state [sheet]), _but_ I can't refute the readability claim. The enum is as practically easy to use as the boolean, and it _feels_ a lot clearer to read than the boolean.